### PR TITLE
Increase timeout when wait for components to be ready in CI

### PR DESCRIPTION
**Pull request by flacatus**

### CODE REVIEW

**Comment by flacatus** [comment]
Sometimes the Jobs are failing with a timeout in build service https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-deployment-periodic/1512219032248389632

File: .ci/oci-e2e-deployment.sh [hunk 1]
```diff
@@ -59,15 +59,15 @@ function createQuayPullSecrets() {
 function waitAppStudioToBeReady() {
     while [ "$(kubectl get applications.argoproj.io ${APPLICATION_NAME} -n ${APPLICATION_NAMESPACE} -o jsonpath='{.status.health.status}')" != "Healthy" ] ||
           [ "$(kubectl get applications.argoproj.io ${APPLICATION_NAME} -n ${APPLICATION_NAMESPACE} -o jsonpath='{.status.sync.status}')" != "Synced" ]; do
-        sleep 1m
+        sleep 2m
```
Comment (Michkov) [inline .ci/oci-e2e-deployment.sh:62]: this is not increasing timeout, just sleep

Comment (Michkov) [reply .ci/oci-e2e-deployment.sh:62]: It's on line 109 -> `timeout --foreground 10m bash -c waitBuildToBeReady`

Comment (flacatus) [reply .ci/oci-e2e-deployment.sh:62]: Lol, my mistake.

**Pull request closed without merging**